dev-docs/publisher-api-reference.md

```diff
@@ -962,4 +977,132 @@ If you define an alias and are using `pbjs.sendAllBids`, you must also set up ad
 + `hb_size_newalias`
 + `hb_deal_newalias`
 
+<a name="module_pbjs.setConfig"></a>
+
+### pbjs.setConfig(options)
+
+{: .alert.alert-info :}
+Added in version 0.27.0
+
+`setConfig` is designed to allow for advanced configuration while reducing the surface area of the public API.  For more information about the move to `setConfig` (and the resulting deprecations of some other public methods), see [the Prebid 1.0 public API proposal](https://gist.github.com/mkendall07/51ee5f6b9f2df01a89162cf6de7fe5b6).
+
+See below for usage examples.
+
+{: .alert.alert-warning :}
+The `options` param object must be JSON - no JavaScript functions are allowed.
+
+Turn on debugging:
+
+{% highlight js %}
+pbjs.setConfig({ debug: true });
+{% endhighlight %}
+
+Set a global bidder timeout:
+
+{% highlight js %}
+pbjs.setConfig({ bidderTimeout: 3000 });
+{% endhighlight %}
+
+Turn on enable send all bids mode:
+
+{% highlight js %}
+pbjs.setConfig({ enableSendAllBids: true })
+{% endhighlight %}
+
+Set the order in which bidders are called to "random":
+
+{% highlight js %}
+pbjs.setConfig({ bidderSequence: "random" })
```

Comment:
`'random'` is now the default bidderSequence ordering as of 0.27.0, would it make since to show the other ordering option in this example, which is `'fixed'`?

Reply:
Great point - will update the example and note the new default, both here and in the docs for the (now-deprecated but still in use) `pbjs.setBidderSequence()`.